src/lib/holds.js

```diff
@@ -1,34 +1,35 @@
 'use strict'
 
 const InsufficientFundsError = require('../errors/insufficient-funds-error')
+const accounts = require('../models/db/accounts')
+const entries = require('../models/db/entries')
 
-function adjustBalance (account, amount, transaction) {
-  const updateSQL = 'UPDATE "L_ACCOUNTS" SET "BALANCE" = "BALANCE" + ? WHERE "NAME" = ?'
+function adjustBalance (accountName, amount, transaction) {
   /* eslint-disable handle-callback-err */
-  return transaction.raw(updateSQL, [amount, account]).catch((error) => {
-    throw new InsufficientFundsError('Sender has insufficient funds.', account)
+  return accounts.adjustBalance(accountName, amount, {transaction})
+  .catch((error) => {
```

Comment:
What do you think about logging this error? It's possible that there could be some database integrity issue (account doesn't exist, etc). I think that should also allow you to remove the `eslint-disable`.